health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/DateRangePicker.js

```
@@ -38,17 +42,28 @@ const DateRangePicker = ({ t, config, onSelect, userType, formData, props }) =>
     return () => document.removeEventListener("mousedown", handleClickOutside);
   }, []);
 
+  const nowEpoch = Date.now();
+  const maxDateForPastCampaigns = new Date(nowEpoch - 24 * 60 * 60 * 1000);
+
   return (
     <div className="custom-daterange-wrapper" ref={ref}>
       <div className="custom-daterange-input" onClick={() => setShowPicker((prev) => !prev)}>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)